manifests/4.5/cluster-logging.v4.5.0.clusterserviceversion.yaml

```diff
@@ -67,33 +67,32 @@ metadata:
             }
           },
           {
-            "apiVersion": "logging.openshift.io/v1alpha1",
-            "kind": "LogForwarding",
+            "apiVersion": "logging.openshift.io/v1",
+            "kind": "ClusterLogForwarder,
```

Comment:
we're changing the kind as part of the API going GA? is there a particular pattern we're following by doing this? wouldn't the `apiVersion` alone be enough to distinguish the two?

Reply:
Thats what we agreed during various discussions regarding clarity and future proofing (we may have a NamespaceLogForwarder in future) You are correct that tthe apiVersion would have been enough.

Reply:
That's what we agreed on after much discussion. One reason is future proofing as someday we may have a NamespacedLogForwarder, so this name is more specific.